docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -1,283 +1,258 @@
 
-When your environment type settings look like the above then go ahead and click `Review and launch`.
+These settings will run the Metabase application using [Docker](https://www.docker.com) under the hood, using the official Metabase Docker image which is [published on Dockerhub](https://hub.docker.com/r/metabase/metabase/).
 
+You can now go ahead and click `Review and launch`.
 
-##### Enabling enhanced health checks
+#### Enabling enhanced health checks
 
 You will need to enable enhanced health checks for your Beanstalk Monitoring.
 
-Click on the `modify` link under the Monitoring section as below.
+Click on the `Modify` link under the Monitoring section as seen here:
+
 ![Elastic Beanstalk Monitoring](images/EBMonitoringSelect.png)
 
 Then make sure enhanced health checks are enabled. This is a free option, unless you later add specific metrics to CloudWatch.
 
 ![Elastic Beanstalk Monitoring Settings](images/EBMonitoringSettings.png)
 
-##### Configuring RDS for Metabase
+#### Configuring RDS for Metabase
 
-To run Metabase in a cloud environment of any kind we highly recommend using an independent database server with high availability such as Amazon RDS.  So for standard deployments we will choose to create an RDS instance with our Elastic Beanstalk application.
+To run Metabase in a cloud environment of any kind we highly recommend using an independent database server with high availability such as Amazon RDS. So for standard deployments we will choose to create an RDS instance with our Elastic Beanstalk application.
 
+_NOTE: It's possible to skip this step if you wish. However, this will force Metabase to use a local H2 database file on your application server, and there will be no way to backup and maintain that database. **When your instance is restarted for any reason you'll lose all your Metabase data**. If you are just doing a quick trial of Metabase that may be okay, but otherwise we recommend against it._
 
-NOTE: it's possible to skip this step if you wish, however this will force Metabase to use a local H2 database file on your application server and there will be no way to backup and maintain that database, so when your instance is restarted for any reason you'll lose all your Metabase data.  If you are just doing a quick trial of Metabase that may be okay, but otherwise we recommend against this.
-
-To set the database password from the Beanstalk template, hit "Review and Launch", and then look for the Database configuration pane as below. It should have a red outline when you first see this page.
+To set the database password from the Beanstalk template, after you've clicked `Review and Launch`, look for the Database configuration pane as below. It should have a red outline when you first see this page. Next, click on the `Modify` link.
```

Comment:
This could be a bit confusing. The "Review and Launch" button is what you click on the original (previous) page to get to this page. Instructing users to click it now, especially after instructions regarding monitoring, could have them search for a button that does not exist on the current page.

Here is the entire main content area of the first page (80% zoom), with the "Review and Launch" button at the bottom: 
![image](https://user-images.githubusercontent.com/23798/66213374-ea90a200-e6bf-11e9-85bc-99948edd130e.png)

Here is the entire main content of the page you get to when clicking "Review and Launch" (80% zoom): 
![image](https://user-images.githubusercontent.com/23798/66213452-17dd5000-e6c0-11e9-90ba-2e8c73be7969.png)